### DISCUSSION TITLE
Find not work

### DISCUSSION THREAD

**Discussion by ricgavira** [dbeaver/dbeaver#8770] [Q&A] [answered]
System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version: 7.0.5.202005171753
- Java version: 1.8.0.202
- Additional extensions

Connection specification:
- I use sqlserver 2012

Describe the problem you're observing:

Steps...

Ctrl + F, insert text e tecle <Enter>

Erro: "String Not Found"

If I use Ctrl + K, OK, String searched.

**Comment by DejfCold** [comment]
@ricgavira Why did you close it? Did you find what's wrong or you're just OK with `Ctrl+K`?

**Comment by ricgavira** [comment]
Eventually the search does not work with Ctrl + F. You try to find a string and accuse it as not found, then you press Ctrl + K and then the string is found.

**Comment by DejfCold** [comment, marked as answer]
@ricgavira Ha! Ctrl + F actually does work. You just have to be connected to a DB. Not sure if that's the actual cause, but it started to work when I connected.

**Reply by mcsilver** [reply]
This was very helpful. Any chance that could be remedied, or is there a practical reason that Ctrl+F doesn't work unless connected to a database?

**Comment by ricgavira** [comment]
Hello. It may be that the connection dropped at that moment, in case it happens again I will inform you again.
